.travis.yml

```diff
@@ -23,15 +21,23 @@ matrix:
   - os: osx
     compiler: gcc
 
+addons:
+  apt:
+    packages:
+      - cmake3
+      - cppcheck
+      - clang-tidy-3.8
+      - libfreetype6-dev
+      - libpng-dev
+
 before_install:
-  - if [ $TRAVIS_OS_NAME == linux ]; then sudo apt-get install -qq cmake3 cppcheck clang-tidy-3.8 libfreetype6-dev libpng-dev; fi
   # libpng is already installed by default
   - if [ $TRAVIS_OS_NAME == osx ]; then brew update && brew install cppcheck freetype; fi
```

Comment:
Is there a way to make syntax for brew the same as for the apt packages in lines 24-31?